test/test_xpath_parser.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: false
+#
+#  Created by Henrik Mårtensson on 2007-02-18.
+#  Copyright (c) 2007. All rights reserved.
+
+module REXMLTests
+  class TestXPathParser < Test::Unit::TestCase
+    def setup
+      @root_element = _make_service_element(["urn:type1", "urn:type2"], ["http://uri"])
+      @root = @root_element.root
+      @element = @root_element.children[0]
+    end
+
+    def _make_service_element(types, uris)
```

Comment:
We don't need to prepend `_` for private method because we aren't using Python:

```suggestion
    def make_service_element(types, uris)
```